### DISCUSSION TITLE
Is there any option to simulate physical mouse ?

### DISCUSSION THREAD

**Discussion by superuser789** [yume-chan/ya-webadb#620] [Q&A]
Thank you for the awesome project. 
There is an option to simulate physical keyboard under screen mirror. But the option to simulate physical mouse is not there.
How to enable it ? I want to use it for termux-X11 to send all mouse movements.

**Comment by CorruptionVR** [comment]
This *was* implemented as a test before in old-adb, but was cut due to low demand/usage and general uninterest, as well as latency that I personally experienced when beta-testing the feature.
if you ant to have a physical mouse though, you could consider using a Bluetooth mouse with multi-connect, such as the M720 Triathlon, which i personally use.

**Reply by superuser789** [reply]
I want to control it using my laptop keyboard & mouse only. Is there any way to make mouse connected to my laptop act as a physical mouse for android.
Thanks.
